crates/runtime/src/http.rs

```diff
@@ -15,12 +15,16 @@ pub enum Error {
 
 type Result<T, E = Error> = std::result::Result<T, E>;
 
-pub(crate) async fn start<F, A>(shutdown_signal: F, bind_address: A) -> Result<()>
+pub(crate) async fn start<F, A>(shutdown_signal: F, bind_address: A, app: &app::App) -> Result<()>
 where
     F: Future<Output = ()> + Send + Sync + 'static,
     A: ToSocketAddrs + Debug,
 {
-    let routes = Router::new().route("/", get(|| async { "Hello, World!" }));
+    let pods_json = serde_json::to_string(&app.spicepods).unwrap();
```

Comment:
This isn't great because any changes made to the pods won't be reflected in the API. I've been unable to figure out how to do this properly because it does not appear that the app reference can be used within the async function. Committing what I have for review for now and going back to learning more Rust basics.

Reply:
Also needs error handling, this is just quick and dirty for now.